apps/molgenis-components/src/client/client.ts

```diff
@@ -166,10 +166,22 @@ const client: IClient = {
         );
       },
       fetchSettings: async () => {
+        if (schemaMetaData === null) {
+          schemaMetaData = await fetchSchemaMetaData(myAxios, schemaNameCache);
+          if (schemaMetaData && !schemaNameCache) {
```

Comment:
if you make fetchSchema cached there is no nee for external null check and cache update for each caller

Reply:
I don't know what you exactly mean but sounds like sensible step. What I did here is a bugfix, I didn't want to refactor the client in this PR